__tests__/app.test.js

```diff
@@ -27,6 +27,36 @@ describe("app", () => {
         });
     });
   });
+  describe("/api", () => {
+    test("200: responds with a status of 200", () => {
+      return request(app).get("/api").expect(200);
+    });
+    test("200: responds with a json object of endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+          const { body } = response;
+          expect(typeof body).toBe("object");
+          const bodyArr = Object.entries(body);
+          bodyArr.forEach((endpointArr) => {
+            const key = endpointArr[0];
+            const value = endpointArr[1];
+            const keyIsReq = key.startsWith(
+              "GET" || "POST" || "PUT" || "DELETE" || "PATCH"
+            );
+            expect(keyIsReq).toBe(true);
+            expect(value).toHaveProperty("description", expect.any(String));
+            expect(value).toHaveProperty("queries", expect.any(Array));
+            expect(value).toHaveProperty(
+              "requestBodyFormat",
+              expect.any(String)
+            );
+            expect(value).toHaveProperty("exampleResponse", expect.any(Object));
+          });
+        });
```

Comment:
A much shorter way of achieving this (that wouldn't need updating when changes are made) is by requiring in the JSON file directly then making making a .toEqual assertion based off of it